spec/spec_helper.rb

```diff
@@ -23,16 +23,17 @@ def hatchet_path(path = "")
   Pathname(__FILE__).join("../../repos").expand_path.join(path)
 end
 
+RUBY_BUILDPACK = Cutlass::LocalBuildpack.new(directory: Pathname(__dir__).join(".."))
+
 Cutlass.config do |config|
-  # Image pinned to fix CI until https://github.com/heroku/buildpacks-ruby/pull/79 lands. 
-  config.default_builder = "heroku/buildpacks:18@sha256:7590c0bc92e574253e44ef3848579869cfeb610b80de093463ef4a7d8de3ae03"
+  config.default_builder = "heroku/buildpacks:20"
```

Comment:
Pleased to see this moving onto Heroku-20, which is the default used by customers.

Since this buildpack didn't previously support Heroku-20 (as it wasn't listed in `buildpacks.toml`), I guess you may also be using Heroku-18 locally?

To change the default locally, run:

```term
pack config default-builder heroku/buildpacks:20
```

Reply:
Done:

```
$ pack config default-builder heroku/buildpacks:20
Builder heroku/buildpacks:20 is now the default builder
```